doc/en/user/source/extensions/css/workshop/index.rst

```diff
@@ -0,0 +1,16 @@
+.. _extensions_css_cartography_styling:
+
+********************
+CSS Styling Workshop
+********************
+
+GeoServer styling can be used for a range of creative effects. This section introduces the `CSS Extension` which can be used to quickly generate SLD files.
```

Comment:
CSS has already been introduced, as this workshop falls at the end of a very long CSS section. Please provide context here on what this workshop will accomplish (and, as already mentioned, how it differs from the tutorial).